scripts/run_android_tests.sh

```diff
@@ -26,10 +26,76 @@ cd android
 export ANDROID_EMULATOR_WAIT_TIME_BEFORE_KILL=$ANDROID_EMULATOR_WAIT_TIME_BEFORE_KILL
```

Comment:
Let's switch this to a node execa script before it gets too much more complicated